install_config/persistent_storage/persistent_storage_csi.adoc

```diff
@@ -0,0 +1,371 @@
+
+[[install-config-persistent-storage-csi-example-deployment]]
+== Example Deployment
+
+Since {product-title} does not ship with any CSI driver installed, this example shows how to deploy a community driver for OpenStack Cinder in {product-title}.
+
+. Create a new project where the CSI components will run and a new service account that will run the components. Explicit node selector is used to run the Daemonset with the CSI driver also on master nodes.
++
+[source,shell]
+----
+# oc adm new-project csi --node-selector=""
+Now using project "csi" on server "https://example.com:8443".
+
+# oc create serviceaccount cinder-csi
+serviceaccount "cinder-csi" created
+
+# oc adm policy add-scc-to-user privileged system:serviceaccount:csi:cinder-csi
+scc "privileged" added to: ["system:serviceaccount:csi:cinder-csi"]
+----
+
+. Apply this YAML file to create the Deployment with the external CSI attacher and provisioner, and DaemonSet with the CSI driver.
++
+[source,yaml]
+----
+# This YAML file contains all API objects that are necessary to run Cinder CSI
+# driver.
+#
+# In production, this needs to be in separate files, e.g. service account and
+# role and role binding needs to be created once.
+#
+# It server as an example how to use external attacher and external provisioner
+# images shipped with {product-title} with a community CSI driver.
+
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: cinder-csi-role
+rules:
+  - apiGroups: [""]
+    resources: ["persistentvolumes"]
+    verbs: ["create", "delete", "get", "list", "watch", "update", "patch"]
+  - apiGroups: [""]
+    resources: ["events"]
+    verbs: ["create", "update", "patch"]
+  - apiGroups: [""]
+    resources: ["persistentvolumeclaims"]
+    verbs: ["get", "list", "watch", "update", "patch"]
+  - apiGroups: [""]
+    resources: ["nodes"]
+    verbs: ["get", "list", "watch", "update", "patch"]
+  - apiGroups: ["storage.k8s.io"]
+    resources: ["storageclasses"]
+    verbs: ["get", "list", "watch"]
+  - apiGroups: ["storage.k8s.io"]
+    resources: ["volumeattachments"]
+    verbs: ["get", "list", "watch", "update", "patch"]
+  - apiGroups: [""]
+    resources: ["configmaps"]
+    verbs: ["get", "list", "watch", "create", "update", "patch"]
+
+---
+
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: cinder-csi-role
+subjects:
+  - kind: ServiceAccount
+    name: cinder-csi
+    namespace: csi
+roleRef:
+  kind: ClusterRole
+  name: cinder-csi-role
+  apiGroup: rbac.authorization.k8s.io
+
+---
+apiVersion: v1
+data:
+  cloud.conf: W0dsb2JhbF0KYXV0aC11cmwgPSBodHRwczovL2V4YW1wbGUuY29tOjEzMDAwL3YyLjAvCnVzZXJuYW1lID0gYWxhZGRpbgpwYXNzd29yZCA9IG9wZW5zZXNhbWUKdGVuYW50LWlkID0gZTBmYTg1YjZhMDY0NDM5NTlkMmQzYjQ5NzE3NGJlZDYKcmVnaW9uID0gcmVnaW9uT25lCg==
+kind: Secret
+metadata:
+  creationTimestamp: null
+  name: cloudconfig
```

Comment:
Add a comment here and tell the storage admin to set cloud.conf to the right value (bug 1580273).

Reply:
I added a note + example how to generate the secret below the yaml file.